src/contexts/__tests__/useSdkOptions.tsx

```diff
@@ -4,11 +4,37 @@ import { EventEmitter2 } from 'eventemitter2'
 
 import { useSdkOptions, SdkOptionsProvider } from '../useSdkOptions'
 import type { NarrowSdkOptions } from '~types/commons'
+import type { StepTypes } from '~types/steps'
 
-const DummyComponent: FunctionComponent = () => {
-  const options = useSdkOptions()
+type DummyProps = {
+  step?: StepTypes
+}
+
+const DummyComponent: FunctionComponent<DummyProps> = ({ step }) => {
+  const [options, { findStep }] = useSdkOptions()
+  const currentStep = findStep(step)
 
-  return <span data-sdk-options={options}>Options</span>
+  return (
+    <span data-sdk-options={options} data-step={currentStep}>
+      Options
+    </span>
+  )
+}
+
+const defaultOptions: NarrowSdkOptions = {
+  token: 'fake-sdk-token',
+  containerId: 'onfido-mount',
+  language: 'fr',
+  userDetails: {
+    smsNumber: '+447495023357',
+  },
+  steps: [
+    { type: 'welcome' },
+    { type: 'document', options: { forceCrossDevice: true } },
```

Comment:
Why does `forceCrossDevice` need to be enabled for the `defaultOptions` with this change?

Reply:
Ah it just a random option. The extra `options` here is to indicate that the logic should work for any arbitrary configuration, not just the default `{ type: 'something' }` ones.